### PR TITLE
[feat] options selector

### LINKED ISSUE #385: option-operator for sellection
have an operator that adds a checjbox `select all` when using a salect multiple

needs a redefinition of what a `ALuOptionSelector` is and how the picker listens to it

### PR DESCRIPTION
**Pull request by lucienbertin**
rebase of #792 fixes #385 

picker

- [x] check that it didnt broke api-operators / user-operators
- [x] make the tree-option-selector also
- [x] use it in department-select

i'll do these in a different PR

- create api-operator that does all (feeder, searcher, pager, selector)
- same but with user, also has a checkbox "former employees"

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://852-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-852

File: packages/ng/libraries/core/src/lib/department/select/input/department-select-input.component.scss [hunk 1]
```diff
@@ -4,3 +4,12 @@
 .lu-select-value {
 	padding-right: 2.5rem;
 }
+
+.option-selector {
+	display: none;
+}
+.mod-multiple {
```
Comment (jeremie-lucca) [inline packages/ng/libraries/core/src/lib/department/select/input/department-select-input.component.scss]: .mod-xxx should never be alone

.lu-picker-header {
  &.mod-multiple {

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-852

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-852

**Pull request merged**